wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/button/CommandGenericHID.java

```diff
@@ -260,6 +261,74 @@ public Trigger axisGreaterThan(int axis, double threshold, EventLoop loop) {
+   *     deadbanded value is non-zero.
+   * @param loop the event loop instance to attach the trigger to.
+   * @return a Trigger instance that is true when the deadbanded axis value is active (non-zero).
+   */
+  public Trigger axisActive(int axis, double deadband, EventLoop loop) {
+    var cache = m_axisGreaterThanCache.computeIfAbsent(loop, k -> new HashMap<>());
+    return cache.computeIfAbsent(
+        Pair.of(axis, deadband),
+        k -> new Trigger(loop, () -> MathUtil.applyDeadband(getRawAxis(axis), deadband) != 0.0));
+  }
+
+  /**
+   * Constructs a Trigger instance that is true when the axis value is active (non-zero) given a
+   * {@code deadband}, attached to {@link CommandScheduler#getDefaultButtonLoop() the default
+   * command scheduler button loop}.
+   *
+   * @param axis The axis to read, starting at 0
+   * @param deadband The value used to deadband the axis value. The trigger returns true if the
+   *     deadbanded value is non-zero.
+   * @return a Trigger instance that is true when the deadbanded axis value is active (non-zero).
+   */
+  public Trigger axisActive(int axis, double deadband) {
+    return axisGreaterThan(axis, deadband, CommandScheduler.getInstance().getDefaultButtonLoop());
+  }
+
+  /**
+   * Constructs a Trigger instance that is true when the axis value is active (non-zero) given a
+   * {@code deadband}, attached to the given loop.
+   *
+   * @param axis The axis to read, starting at 0
+   * @param deadband The value used to deadband the axis value. The trigger returns true if the
+   *     deadbanded value is non-zero.
+   * @param maxMagnitude The maximum magnitude of the input. Can be infinite.
+   * @param loop the event loop instance to attach the trigger to.
+   * @return a Trigger instance that is true when the deadbanded axis value is active (non-zero).
+   */
+  public Trigger axisActive(int axis, double deadband, double maxMagnitude, EventLoop loop) {
+    var cache = m_axisGreaterThanCache.computeIfAbsent(loop, k -> new HashMap<>());
+    return cache.computeIfAbsent(
+        Pair.of(axis, deadband),
+        k ->
+            new Trigger(
+                loop,
+                () -> MathUtil.applyDeadband(getRawAxis(axis), deadband, maxMagnitude) != 0.0));
+  }
+
+  /**
+   * Constructs a Trigger instance that is true when the axis value is active (non-zero) given a
+   * {@code deadband}, attached to {@link CommandScheduler#getDefaultButtonLoop() the default
+   * command scheduler button loop}.
+   *
+   * @param axis The axis to read, starting at 0
+   * @param deadband The value used to deadband the axis value. The trigger returns true if the
+   *     deadbanded value is non-zero.
+   * @param maxMagnitude The maximum magnitude of the input. Can be infinite.
+   * @return a Trigger instance that is true when the deadbanded axis value is active (non-zero).
+   */
+  public Trigger axisActive(int axis, double deadband, double maxMagnitude) {
```

Comment:
The max magnitude doesn't affect whether the result is non-zero, so this overload doesn't offer any benefits.